conf/modules.config

```diff
@@ -133,6 +134,7 @@ process {
 
     withName: '.*:FASTQ_ALIGN_BOWTIE2:BOWTIE2_ALIGN' {
         ext.args = '--no-unal'
+        ext.args2 = '-f 3'
```

Comment:
Is` ext.args2` needed here? This would not work?

`--no-unal -f 3`

Reply:
`ext.args2` defines the parameter for [samtools](https://github.com/genomic-medicine-sweden/meta-val/blob/37ddce2df4b41a66d8e89e39e1ce02e7b07fe65c/modules/nf-core/bowtie2/align/main.nf#L64) and `ext.arg` defines the parameters for [bowtie2](https://github.com/genomic-medicine-sweden/meta-val/blob/37ddce2df4b41a66d8e89e39e1ce02e7b07fe65c/modules/nf-core/bowtie2/align/main.nf#L62C10-L62C14)